qa/automation/CareerPortal.Tests.Scenarios/1.Send your CV/Send CV.feature

```diff
@@ -0,0 +1,275 @@
+	Then user sees that the line under the text field is highlighted in green
+
+	Examples: 
+	| Field     | Description |
+	| "Name"    | O O         |
+	| "Email"   | f@e.ru      |
+	| "Message" | 1213        |
+	
+	
+
+@5
+@Placeholders
+
+Scenario Outline: 1.1.12. Checking the placeholders in  text fields
+    Given user sees the plceholder <Placeholder> in first text field 
+	When user clicks in <Text field> text field and fill in with ""
+	And user clicks on other place
+	Then placeholder <Placeholder> appears back
+
+	Examples: 
+	| Placeholder | Text field |
+	| Name*       | Name       |
+	| Email*      | Email      |
+	| Message     | Message    |
+
+Scenario Outline: 1.1.13. Checking the disappearing of placeholders in  text fields
+    Given user sees the plceholder <Placeholder> in first text field 
+	When user clicks in <Text field> text field and fill in with " "
+	And user clicks on other place
+	Then placeholder <Placeholder> disappears
+
+	Examples: 
+	| Placeholder | Text field |
+	| Name*       | Name       |
+	| Email*      | Email      |
+	| Message     | Message    |
+
+@6
+@AttachCVbutton
+
+Scenario Outline: 1.1.14. Attaching a valid document 
+	When user clicks on "Attach CV" button
+	And user attaches a file in <format> format weighing less or equal 10 Mb with the name <name>
+    Then user sees that under "Attach CV" button attached document <name> appeared 
+
+	Examples: 
+	| format | name      |
+	| PDF    | name.pdf  |
+	| DOC    | name.doc  |
+	| DOCX   | name.docx |
+	| RPF    | name.rpf  |
+
+
+
+Scenario Outline: 1.1.15. Attaching invalid document 
+    When user clicks on "Attach CV" button
+   	And user attaches a file in <format> format weighing less or equal 10 Mb with the name <name>
+    Then user sees a warning message "Something went wrong Invalid file format. Please try PDF, DOC, DOCX or RTF." and "OK" button
+	And user clicks on "OK" button
+	And user sees that <name> did not attach
+
+
+	Examples: 
+	| format | name     |
+	| TXT    | name.txt |
+	| GIF    | name.gif |
+	| XLS    | name.xls |
+	| XML    | name.xml |
+
+Scenario Outline: 1.1.16. Attaching heavy valid document 
+	When user clicks on "Attach CV" button
+	And user attaches a file in <format>  weighing more than 10 Mb with the name <name>
+    Then user sees a warning message "Something went wrong Max file size 10 Mb" and "OK" button
+	And user clicks on "OK" button
+	And user sees that <name> did not attach
+
+	Examples: 
+	| format | name      |
+	| PDF    | name.pdf  |
+	| DOC    | name.doc  |
+	| DOCX   | name.docx |
+	| RPF    | name.rpf  |
+
+@7
+@SendCVbutton
+
+Scenario: 1.1.17. Consent to data processing
+	When user clicks on checkbox near "I agree to the personal data processing"
+	Then user sees that chech mark appears inside the box to indicate an affirative choice
+
+#здесь не перепбирались все варианты, тут pairwise
+Scenario Outline: 1.1.18. Unsuccessful attempts to send CV
```

Comment:
There are multiple "When" steps, that should be avoided according to the BDD cardinal rule.
In comment line 264 I have suggested how this issue could be fixed.